subjects/highest/README.md

```diff
@@ -33,9 +41,6 @@ Here is a program to test your function.
 use highest::*;
 
 #[derive(Debug)]
```

Comment:
You should remove this `derive` directive (and maybe also the extra empty line behind it).